test/suite/test_verify_disagg.py

```diff
@@ -138,3 +137,42 @@ def test_verify_disagg(self):
         self.verify([self.session])
         # FIXME-WT-14700: remove ignore after freeing root pages is addressed.
         self.ignoreStdoutPattern("Mismatch in page IDs")
+
+    def test_verify_leader_no_table(self):
+        # Layered table does not exist, expect ENOENT
+        self.verify([self.session], errno.ENOENT)
+
+    def test_verify_follower_no_metadata(self):
+         # Create a layered table on the leader
+        self.session.create(self.uri, self.table_cfg)
+        # Verify the empty leader's table
+        self.verify([self.session])
+
+        # Create a follower
+        self.create_follower()
+        # The follower has not picked up its first checkpoint. The follower will not recognize
+        # the layered URI while in this transient state. Expect ENOENT.
+        self.verify([self.session_follow], errno.ENOENT)
```

Comment:
Can you make it pick up a checkpoint so we show verify first returns ENOENT and then 0 after picking a checkpoint?